### DISCUSSION TITLE
模拟器选型

### DISCUSSION THREAD

**Discussion by ChnMig** [TongchengOpenSource/AppScan#29] [Polls]
请投票给您日常使用或希望支持的模拟器
我们可能会优先适配投票数最多的模拟器
投票截止时间为 2023/10/20

**Comment by ChnMig** [comment]
雷电模拟器9本地开发环境已经适配成功 🥇 
MuMu模拟器12因为没有看到开启USB调试选项, 而且ADB无法找到模拟器设置暂停适配 👎

**Comment by ChnMig** [comment]
投票已结束
MuMu浏览器因为USB调试无法找到的问题暂停适配
雷电模拟器9已经完成适配, 用户可以下载pre版本试用 https://github.com/TongchengOpenSource/AppScan/releases/tag/2.0.0
夜神模拟器正在考虑适配

**Comment by sanan2015** [comment]
MuMu模拟器需要在“设置”-“关于手机”-“版本号”，点三次进入开发者模式，之后在“设置”-“系统”里可以看到开发者选项。
MuMu adb可以直接连接，端口是7555，可以看看开发者文档：https://mumu.163.com/help/20230214/35047_1073151.html
